next.config.js

```diff
@@ -27,10 +27,15 @@ const sentryWebpackPluginOptions = {
   // Additional config options for the Sentry Webpack plugin. Keep in mind that
   // the following options are set automatically, and overriding them is not
   // recommended:
-  //   release, url, org, project, authToken, configFile, stripPrefix,
-  //   urlPrefix, include, ignore
+  //   release, url, configFile, stripPrefix, urlPrefix, include, ignore
 
   silent: true, // Suppresses all logs
+  project: process.env.SENTRY_PROJECT,
+  org: process.env.SENTRY_ORG,
+  authToken: process.env.SENTRY_AUTH_TOKEN,
```

Comment:
these values are set by default when they're included in sentry.properties and the git ignored ".sentryclirc". now that they're removed from those files, this config needs to be updated on where to find them.